web/src/message_edit.js

```diff
@@ -434,12 +436,14 @@ function create_copy_to_clipboard_handler($row, source, message_id) {
 }
 
 function edit_message($row, raw_content) {
+    // NOTE: It is possible that the message list hasn't been rendered yet,
+    // so we can't query DOM directly, keep all the selector queries and events
+    // to `$row` or `$form`.
```

Comment:
Tweaked this to serve as a proper docstring for the function.

``` diff
diff --git a/web/src/message_edit.js b/web/src/message_edit.js
index b072aa3d12..d703aad990 100644
--- a/web/src/message_edit.js
+++ b/web/src/message_edit.js
@@ -436,9 +436,11 @@ function create_copy_to_clipboard_handler($row, source, $message_edit_content) {
 }
 
 function edit_message($row, raw_content) {
-    // NOTE: It is possible that the message list hasn't been rendered yet,
-    // so we can't query DOM directly, keep all the selector queries and events
-    // to `$row` or `$form`.
+    // Open the message-edit UI for a given message.
+    //
+    // Notably, when switching views, this can be called for a row
+    // that hasn't been added to the DOM yet so, keep all the selector
+    // queries and events to operate on `$row` or `$form`.
     assert(message_lists.current !== undefined);
     const message = message_lists.current.get(rows.id($row));
     $row.find(".message_reactions").hide();
```